### DISCUSSION TITLE
Keyboard Lightning Mode Heatmap & Ambient Don't Work occasionally

### DISCUSSION THREAD

**Discussion by ZhuGene** [seerge/g-helper#2766] [Q&A]
My Laptop Model is Rog Zephyrus G14 GA403UV，as the title, Keyboard Lightning Mode Heatmap & Ambient sometimes work and sometimes don't, and I don't know what the problem is. 
![屏幕截图 2024-06-24 100912](https://github.com/seerge/g-helper/assets/123133054/8766bc34-26ab-453f-80f6-cd64822f5666)

**Comment by seerge** [comment]
@ZhuGene I also don't know what happens specifically on your device, as you have [completely ignored issue template](https://github.com/seerge/g-helper/issues/new/choose) and didn't include logs or clear reproduction scenario.

Make sure that you have disabled ``Dynamic Lighting`` in Windows Settings and also don't have any other apps or AC leftovers running and interfering.

**Reply by ZhuGene** [reply]
![快照1](https://github.com/seerge/g-helper/assets/123133054/18014929-a418-4b97-ad50-c2ac47d7cb3b)
Yes, I have disabled Dynamic Lightning in Windows Setting. I used "Armoury Crate Uninstall Tool" to uninstall Armoury Crate, and in G-Helper I stopped all ASUS services. 
Sorry I didn't know that I should use issue template to report my issue, I will do this, thank you for your reply.
